Code/GraphMol/ForceFieldHelpers/MMFF/AtomTyper.h

```diff
@@ -217,7 +217,6 @@ RDKIT_FORCEFIELDHELPERS_EXPORT bool areAtomsInSameRingOfSize(
     const ROMol &mol, const unsigned int ringSize, const unsigned int numAtoms,
     ...);
 RDKIT_FORCEFIELDHELPERS_EXPORT unsigned int sanitizeMMFFMol(RWMol &mol);
-RDKIT_FORCEFIELDHELPERS_EXPORT void setMMFFAromaticity(RWMol &mol);
```

Comment:
Rather than completely removing this, which may break others' code, please mark it as deprecated (and add a note in the `Deprecated code` section of README.md) and call the `MolOps` version from it.

Reply:
Done! Please check if the deprecation markings are ok, tried my best to copy how they are done elsewhere